build_centos.dockerfile

```diff
@@ -0,0 +1,46 @@
+# https://github.com/ghdl/ghdl/issues/1484
+# https://github.com/ghdl/ghdl/issues/1118
+
+#
+# Centos 7
+#
+
+FROM centos:centos7
```

Comment:
Please, use `ARG IMAGE="centos:centos7"` above, and `FROM $IMAGE`.

Reply:
Done. I used centos:7, which is also valid (instead of centos:centos7)